package.json

```diff
@@ -144,7 +146,8 @@
   "resolutions": {
     "gauge/strip-ansi": "6.0.1",
     "wide-align/string-width": "4.2.3",
-    "cliui/wrap-ansi": "7.0.0"
+    "cliui/wrap-ansi": "7.0.0",
+    "**/sucrase": "getsentry/sucrase#es2020-polyfills"
```

Comment:
Gotta admit, I had no idea that one can simply specify a repo/branch name here. Nice!

Reply:
yeah, it required us to ensure that the dist folder is actually checked in on that branch, but IMHO this appears easier than to publish stuff from there etc 😅